src/vector/defn.rs

```diff
@@ -122,13 +122,13 @@ impl<'a> Field<'a> {
     /// Get the name of this field.
     pub fn name(&'a self) -> String {
         let rv = unsafe { gdal_sys::OGR_Fld_GetNameRef(self.c_field_defn) };
-        _string(rv)
+        _string(rv).unwrap_or_default()
     }
 
     /// Get the alternative name (alias) of this field.
     pub fn alternative_name(&'a self) -> String {
         let rv = unsafe { gdal_sys::OGR_Fld_GetAlternativeNameRef(self.c_field_defn) };
```

Comment:
`rv` is owned by `c_field_defn` here so we don't need to free it?

Reply:
Yes, owned by `OGRFieldDefn`.